catalog/transformer/catalog-transformer-overlay/src/main/java/ddf/catalog/transformer/OverlayMetacardTransformer.java

```diff
@@ -59,10 +58,20 @@
   private final BiFunction<Metacard, Map<String, Serializable>, Optional<BufferedImage>>
       imageSupplier;
 
+  private final BiFunction<Metacard, Map<String, Serializable>, Optional<Geometry>>
+      boundarySupplier;
+
   public OverlayMetacardTransformer(
-      BiFunction<Metacard, Map<String, Serializable>, Optional<BufferedImage>> imageSupplier) {
+      BiFunction<Metacard, Map<String, Serializable>, Optional<BufferedImage>> imageSupplier,
+      BiFunction<Metacard, Map<String, Serializable>, Optional<Geometry>> boundarySupplier) {
```

Comment:
❗️ does this break Alliance?  https://github.com/codice/alliance/blob/master/catalog/imaging/imaging-transformer-nitf/src/main/resources/OSGI-INF/blueprint/blueprint.xml#L176

Reply:
I don't think so since there's still a constructor that takes only an image supplier and the default boundary supplier logic is the same as before.